mesonbuild/scripts/__init__.py

```diff
@@ -12,10 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from pathlib import Path
```

Comment:
I'd prefer to explicitly use PurePath instead of Path.

The only difference between the two is that a concrete `Path()` exposes a bunch of additional, but broken, I/O methods that aren't even used.